packages/block-library/src/post-featured-image/style.scss

```diff
@@ -6,4 +6,17 @@
 		max-width: 100%;
 		height: auto;
 	}
+	&.is-used-as-cover {
+		position: relative;
+		min-height: 300px;
+		background-position: center center;
+		background-size: cover;
+		a {
```

Comment:
@jasmussen is there a better way to achieve full height/width link with no content?

Reply:
would it make more sense to just wrap the image in an a tag?